docs/assets/stylesheets/extra.css

```
@@ -0,0 +1,842 @@
+  font-weight: 600;
+  color: var(--md-default-fg-color);
+  margin: 0 0 0.15rem 0;
+  line-height: 1.2;
+}
+
+.md-typeset .ck-control-loop__step p {
+  margin: 0;
+  color: var(--md-default-fg-color--light);
+  line-height: 1.3;
+  font-size: 0.6875rem;
+  word-wrap: break-word;
```

Stylelint (17.5.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)